Add server login

paytonshaltis opened the pull request:
Using Express Sessions and a simple 'users' table in the SQLite database, a user can now create an account, log in, and log out of the application using a simple API.

These changes also create a directory structure that can be used to keep the rest of the application consistent during development.

sderacy commented:
Interesting. It looks like the stuff you did created some webpages as well. I have some code on my end where I was working on the landing page so I'll have to touch base with you on that.

Looks good to me though

paytonshaltis commented:
> Interesting. It looks like the stuff you did created some webpages as well. I have some code on my end where I was working on the landing page so I'll have to touch base with you on that.
> 
> Looks good to me though

Yeah, I mainly added these for the basic login and signup forms, and also because pages have to follow a certain directory structure to work. I didn't add any styles or elaborate in the HTML, I just added enough to facilitate login and signup. When you add pages and styles, you can just put them in those basic empty file that I added. I can explain more about the structure of the project when we meet up at some point.
